reference/gdscript.rst

```diff
@@ -713,33 +713,34 @@ Basic syntax:
 
 **Crash-course for people who are familiar to switch statements**:
 
-1) replace ``switch`` by ``match``
+1) replace ``switch`` with ``match``
 2) remove ``case``
-3) remove any ``break`` s. If you don't want to ``break`` by default you can use ``continue`` to get a fallthrough.
+3) remove any ``break``'s. If you don't want to ``break`` by default you can use ``continue`` for a fallthrough.
 4) change ``default`` to a single underscore.
 
 
 **Control flow**:
 
-The patterns get matched from top to bottom.
-If a pattern matches the corresponding block gets executed. After that the execution will continue after the ``match`` statement.
-If you want to have a fallthrough you can use ``continue`` to stop execution in the current block and check the next one.
+The patterns are matched from top to bottom.
+If a pattern matches, the corresponding will be block executed. After that, the execution continues below the ``match`` statement.
+If you want to have a fallthrough you can use ``continue`` to stop execution in the current block and check the ones below it.
 
 
 
 
 There are 6 pattern types:
 
 - constant pattern
-    constant primitives like numbers and strings ::
+    constant primitives, like numbers and strings ::
     
         match x:
             1:      print("I'm number one!")
```

Comment:
Can we change that to "We're number one!" ? Because of... cultural reasons. *cough*